docs/source/conf.py

```diff
@@ -111,7 +111,7 @@
     'extra_footer': '<p align="right"><a href="https://www.intel.com/content/www/us/en/privacy/intel-cookie-notice.html">Cookies</a></p>'
 }
 
-html_theme_options = { 
+html_theme_options = {
```

Comment:
This is also addressed in the https://github.com/oneapi-src/oneDAL/pull/2428, so maybe you can skip it